.github/workflows/release-create.yml

```diff
@@ -103,6 +103,17 @@ jobs:
         | Before release Git HEAD    | \`$(git rev-parse HEAD)\` |
         !
 
+    - name: Update README.md + SECURITY.md + server-upgrade.md files
```

Comment:
nit: imo the name is bound to get outdated with the script so i would use a more generic one maybe